src/LibrarianDao.java

```diff
@@ -0,0 +1,47 @@
+import java.sql.*;
+public class LibrarianDao {
+
+	
+	public static int save(String name,String password,String email,String address,String city,String contact){
+		int status=0;
+		try{
+			Connection con=DB.getConnection();

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *con*. The resource is referenced in statements at the following line: 9. The resource closure statement is at line: 17. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by Connection.prepareStatement. Either a) close *con* in a try-finally block or b) close the resource by declaring *con* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).